frontend/src/metabase/public/components/EmbedFrame.jsx

```diff
@@ -43,12 +43,13 @@ class EmbedFrame extends Component {
     } = this.props;
     const { innerScroll } = this.state;
 
-    const showFooter = !MetabaseSettings.hideEmbedBranding() || actionButtons;
-
-    const { bordered, titled, theme, hide_parameters } = {
+    const { bordered, titled, theme, hide_parameters, show_download_button } = {
       ...DEFAULT_OPTIONS,
       ...parseHashOptions(location.hash),
     };
+    const showFooter =
```

Comment:
I think the footer should be hidden when there's no button otherwise it'll look weird.
#### Footer with button
![image](https://user-images.githubusercontent.com/1937582/175021095-0cfb78be-8489-4eaa-b9fd-72e734c38b44.png)

#### Footer without button
![image](https://user-images.githubusercontent.com/1937582/175021144-a8b15aaa-36af-4809-8700-4acb7abe6bf8.png)